test/setup-kubernetes.sh

```diff
@@ -267,7 +265,17 @@ cat ${kubeadm_config_file}
 # 5.0.9-301.fc30.x86_64).
 kubeadm_args="$kubeadm_args --ignore-preflight-errors=SystemVerification"
 
-kubeadm_args_init="$kubeadm_args_init --config=$kubeadm_config_file"
+# Migrate to newer config version incase if we are using outdated config schema.
+# `kubeadm config migrate` vanishing the 'KubeletConfiguration' while doing this.
+# So, append to post-migarted configuraiton.
```

Comment:
@avalluri Do you want to fix spelling mistakes or should we ignore them?

This here and the commit message both have mistakes.

Reply:
I am fixing them part of fixing <=v1.19 failures